src/Task1/SearchForm.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+
+const SearchForm = () => {
+  const [term, setTerm] = useState("");
+  return (
+    <div>
+      <div style={{ display: "flex", justifyContent: "center" }}>
+        <form
+          style={{
+            width: "50%",
+            display: "flex",
+            justifyContent: "space-around",
+          }}
+        >
+          <input
+            onChange={(e) => {
+              setTerm(e.target.value);
+            }}
+            style={{
+              background: "#323232",
+              outline: "none",
+              border: "none",
+              width: "70%",
+              height: "30px",
+              borderRadius: "3px",
+              color: "#FFFFFF",
+              paddingLeft: "10px",
+            }}
+            placeholder="What do you want to watch?"
+            type="search"
+          ></input>
+          <button
+            onClick={(e) => {
+              e.preventDefault();
+              console.log(term);
+            }}
+            style={{
+              backgroundColor: "#F65261",
+              height: "30px",
+              borderRadius: "2px",
+              color: "#FFFFFF",
+              outline: "none",
+              border: "none",
+              width: "25%",
+            }}
```

Comment:
Please, apply styles using css classes instead of inline styles. Inline styles are generally considered as bad practices. Here are some reasons why:

- Slower in terms of performance
- Aren't scalable well, as it's harder to re-use and compose them, and are mixed with other code thus breaking separation of concerns principle
- Have some functionality limitations (e.g. pseudo-classes like `:hover

This comment is related to all places where you use inline styles. Update all of them to use classes, unless you have reasons to stay with inline styles for some cases.

Reply:
I fixed